docs/book/src/cronjob-tutorial/testdata/project/config/default/kustomization.yaml

```diff
@@ -36,7 +36,7 @@ patchesStrategicMerge:
 
 # [WEBHOOK] To enable webhook, uncomment all the sections with [WEBHOOK] prefix including the one in
 # crd/kustomization.yaml
-#- manager_webhook_patch.yaml
+- manager_webhook_patch.yaml
```

Comment:
Since you've already made the changes in the file `hack/docs/internal/cronjob-tutorial/generate_cronjob.go`, do we really need the change here in this file? Asking because this will by overwritten by `make generate` (and hence make generate-docs) as mentioned by @camilamacedo86 .

Reply:
As I understand, `make generate` will overwrite the file `docs/book/src/cronjob-tutorial/testdata/project/config/default/kustomization.yaml`, right? 

If so, then do we explicitly need to make this change here? Else, this can be skipped. Thanks!

Reply:
Please take a look at https://github.com/kubernetes-sigs/kubebuilder/actions/runs/5059859019/jobs/9085111581?pr=3427 for the `Verify docs` session